### DISCUSSION TITLE
How to clear the Agent session

### DISCUSSION THREAD

**Discussion by nandishmb** [mindsphere/mindconnect-nodejs#315] [General]
1. Run the test demo with one agentconfig.json in a machine (1st attempt) - works fine - agent onboarded
2. Run the same test with different agentconfig.json in same machine (no other change)  - invalid token error - new agent not onboarded

How to clear the session?

**Comment by sn0wcat** [comment]
https://opensource.mindsphere.io/docs/mindconnect-nodejs/troubleshooting.html#tab1anchor5

How to reset agent configuration
If you have problems with your agent:

1.Stop the agent.
2.Move or delete the content of the .mc folder (the json files with configuration and authentication settings).
3.Offboard the agent.
4.Create new settings for the mindconnect library.
